src/components/Expenses/ExpenseFilter.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./ExpenseFilter.css";
+
+type PropsType = {
+  selectedYear: string;
+  showFilteredByYear: (selectedYear: string) => void;
+};
+
+const ExpenseFilter = ({ showFilteredByYear, selectedYear }: PropsType) => {
+  const onFilterChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
There is no point having the whole `e` here. Destructure it and get the only think that you use, which is `currentTarget`.